docs/4-user-guide/4-vscode.md

```diff
@@ -0,0 +1,26 @@
+# Getting started - VS Code
+
+Zarf uses an internally defined [schema](https://github.com/defenseunicorns/zarf/blob/master/zarf.schema.json) for its configuration files. This schema is used to validate the configuration files before they are used to build a Zarf package.
```

Comment:
This is a little awkwardly worded.  It is internal/bespoke to Zarf, but the schema is used for more than validation, it is how Zarf marshals/unmarshals the yaml.  Something like this might be clearer:

```
Zarf uses its own [schema](https://github.com/defenseunicorns/zarf/blob/master/zarf.schema.json) to define its configuration files. This schema is used to describe package configuration options and can be used to validate the configuration files before they are used to build a Zarf package.
```